tests/src/pages/grafana.py

```diff
@@ -8,22 +10,25 @@ def login(page: Page, username: str, password: str):
         - `username (str)`: The username
         - `password (str)`: The password
     """
+    time.sleep(1)
```

Comment:
Without sleep the login form was sometimes not available yet and wait_for cannot be used because login is only necessary for the first test. I'm not proud of the sleep - but on my machine this increased the stability.

Reply:
:+1: